.github/workflows/backend.yml

```diff
@@ -6,32 +6,30 @@ on:
   push:
     branches: [main]
 
-defaults:
-  run:
-    working-directory: backend
-
 jobs:
   backend-ci:
     name: Run Backend Tests and Linter
     runs-on: ubuntu-latest
 
     steps:
       - name: Check out Git repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Set up Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version: latest
```

Comment:
Alright - how about we change it to `lts/*`? Looking at the documentation for the [setup-node action](https://github.com/actions/setup-node#supported-version-syntax) this will install the latest LTS version of node (which is currently [v16.15.1](https://nodejs.org/en/)). I think this is a reasonable tradeoff between ensuring our CI node version stays reasonably up to date while retaining a stable version.

Reply:
yeah that sounds pretty fair

Reply:
Changed in 2cd9fa4e